data/data/powervs/cluster/power_network/variables.tf

```diff
@@ -29,3 +29,8 @@ variable "cloud_conn_name" {
   description = "The name of a pre-created Power VS Cloud connection."
   default     = ""
 }
+
+variable "machine_cidr" {
+  type        = string
+  description = "The machine network (IPv4 only)"
+}
```

Comment:
nit: This passes tf tests locally, but adding the newline at the end is the suggestion